### ISSUE TITLE
implement queries for Euclid and Spherex 

### ISSUE THREAD

**Issue by jkrick** [nasa-fornax/fornax-demo-notebooks#346]
Does Mock data exist?
When will we get real Euclid spectroscopy in the format and served from the location that users will actually see?

**Comment by jkrick** [comment]
update from Troy on Oct 3:
"Update on Euclid data availability from IRSA backend mtg today:
Euclid should be sending us a bunch of data in November. IRSA’s planning have our backend services set up to serve it through a private web sever for testing sometime in January. I’m not exactly sure how all the connections work but I think we ought to be able to access it at that point using pyvo or astroquery, though probably will need to be logged in to IPAC VPN or on (e.g.) IRSA science platform to do it."

**Comment by bsipocz** [comment]
We have now a bunch of euclid notebooks at IRSA tutorials. Should we close this here?

**Comment by jkrick** [comment]
no, I was hoping to one day add it to the spectroscopy notebook for added science benefit.  But now that the examples already exist, I am hoping that is a quick process.
